### PR TITLE
Feature/update readme

### PR DESCRIPTION
**Pull request by vinhnglx**
@duykhoa  https://github.com/FancyBlog/blackhulk/blob/401d3e2630916cb9fa6d647cfd90da85f3d193fb/README.md

solve #14

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Tran Duy Khoa
```
Comment (duykhoa) [inline LICENSE.md]: And Vinh Nguyen??

File: README.md [hunk 2]
```diff
@@ -2,4 +2,48 @@
 
 [![Build Status](https://travis-ci.org/FancyBlog/blackhulk.svg?branch=develop)](https://travis-ci.org/FancyBlog/blackhulk)
 
-Blog engine for developer.
+BlackHulk is a awesome blog engine for developer. Your posts will be saved on a GitHub repository and BlackHulk will display on a web browser.
```
Comment (duykhoa) [inline README.md]: .. is **an** awesome..

File: README.md [hunk 3]
```diff
@@ -2,4 +2,48 @@
...
+
+# Installation
+
+```
+$ ./scripts/setup
+```
+
+# How it works
+
+- Firstly, let's create new repository on your GitHub account. Blog posts are open issues labeled "Blog".
```
Comment (duykhoa) [inline README.md]: It's mean we need to add a label called "Blog", right?

File: README.md [hunk 4]
```diff
@@ -2,4 +2,48 @@
...
+
+1. Fork it ( https://github.com/FancyBlog/blackhulk/fork )
+2. Create your feature branch (git checkout -b my-new-feature)
+3. Commit your changes (git commit -am 'Add some feature')
+4. Push to the branch (git push origin my-new-feature)
+5. Create a new Pull Request
+
+# Copyright
+
+Copyright © 2016 Duy Khoa – Released under MIT License
```
Comment (duykhoa) [inline README.md]: Vinh nguyen?

File: README.md (hunk 2 above)
Comment (vinhnglx) [inline README.md]: oops

File: README.md (hunk 3 above)
Comment (vinhnglx) [inline README.md]: yes

File: LICENSE.md (hunk 1 above)
Comment (vinhnglx) [inline LICENSE.md]: okay ....

**Pull request merged**